src/browser/tab/mod.rs

```diff
@@ -185,17 +187,31 @@ impl<'a> Tab {
             .map_err(|e| e.into())
     }
 
+    pub fn wait_for_elements(&'a self, selector: &'a str) -> Result<Vec<Element<'a>>, Error> {
+        debug!("Waiting for element with selector: {}", selector);
+        util::Wait::with_timeout(Duration::from_secs(15))
+            .until(|| {
+                if let Ok(elements) = self.find_elements(selector) {
+                    if elements
+                        .iter()
+                        .all(|element| element.get_midpoint().is_ok())
```

Comment:
Why is this here?

Reply:
It really shouldn't be! Removed from here and from `wait_for_element`.